pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -283,6 +283,60 @@ func (s *USMSuite) TestTLSClassification() {
 	}
 }
 
+func (s *USMSuite) TestTLSClassificationAlreadyRunning() {
+	t := s.T()
+
+	cfg := testConfig()
+	cfg.ProtocolClassificationEnabled = true
+	cfg.BPFDebug = true
+
+	if !classificationSupported(cfg) {
+		t.Skip("TLS classification platform not supported")
+	}
+
+	serverAddr := net.JoinHostPort("localhost", httpsPort)
+	portAsValue, err := strconv.ParseUint(httpsPort, 10, 16)
+	require.NoError(t, err)
+
+	_ = testutil.HTTPPythonServer(t, serverAddr, testutil.Options{
+		EnableKeepAlive: false,
+		EnableTLS:       true,
+	})
+
+	client := &nethttp.Client{
+		Transport: &nethttp.Transport{
+			TLSClientConfig: &tls.Config{InsecureSkipVerify: true},
+		},
+	}
+
+	// Helper to make a request to the server, and discard the response.
+	makeRequest := func() {
+		resp, err := client.Get(fmt.Sprintf("https://%s/200/test", serverAddr))
+		require.NoError(t, err)
+
+		_, _ = io.Copy(io.Discard, resp.Body)
+		_ = resp.Body.Close()
+	}
+
+	tr := setupTracer(t, cfg)
+	require.NoError(t, tr.ebpfTracer.Pause())
+
+	makeRequest()
+	require.NoError(t, tr.ebpfTracer.Resume(), "enable probes")
+	makeRequest()
```

Comment:
I think you can do the following as a simpler version

```suggestion
	makeRequest()
	tr := setupTracer(t, cfg)
	// wait to allow tracer properly setup
	makeRequest()
```